From a43774d4dd619fee7a60e888681f4f899bd0def1 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Sat, 16 Aug 2014 19:02:06 +0800
Subject: [PATCH] egl-util: remove GL header

EGL util should not in any way depend on the headers of a particular
client library.

In fact, this breaks compilation on pure OpenGL ES systems, since they
do not have OpenGL headers installed (<GL/gl.h>).

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
---
 tests/egl/egl-util.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/egl/egl-util.h b/tests/egl/egl-util.h
index f5c74fd..27d2926 100644
--- a/tests/egl/egl-util.h
+++ b/tests/egl/egl-util.h
@@ -12,7 +12,6 @@
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 #include <X11/keysym.h>
-#include <GL/gl.h>
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 
-- 
2.1.0.rc2.206.gedb03e5

